### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by marcin-krolik**
Updates documentation

Summary of changes:
- capitalized Snap
- updated links
- updated example

@intelsdi-x/plugin-maintainers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,37 +22,37 @@ It's used in the [snap framework](http://github.com:intelsdi-x/snap).
 * [golang 1.5+](https://golang.org/dl/)
```
Comment (katarzyna-z) [inline README.md]: Please change golang version to 1.6+

File: README.md [hunk 2]
```diff
@@ -22,37 +22,37 @@ It's used in the [snap framework](http://github.com:intelsdi-x/snap).
...
-#### Download load plugin binary:
-You can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://githubYou can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://github.com/intelsdi-x/snap/releases) page. Download the plugins package from the latest release, unzip and store in a path you want `snapd` to access.
+#### Download the plugin binary:
+
+You can get the pre-built binaries for your OS and architecture from the plugin's [GitHub Releases](https://github.com/intelsdi-x/snap-plugin-collector-load/releases) page. Download the plugin from the latest release and load it into `snapd` (`/opt/snap/plugins` is the default location for Snap packages).
 
 #### To build the plugin binary:
-Fork https://github.com/intelsdi-x/snap-plugin-collector-load  
+
+Fork https://github.com/intelsdi-x/snap-plugin-collector-load
```
Comment (katarzyna-z) [inline README.md]: This line is joined with the next line, please add a new line after this sentence.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
-# snap collector plugin - load
+# Snap collector plugin - load
 This plugin collects metrics from /proc/loadavg kernel interface about load average figures giving the number of jobs in the run queue (state R) or waiting for disk I/O (state D) averaged over 1, 5, and 15 minutes.  
 
-It's used in the [snap framework](http://github.com:intelsdi-x/snap).
+It's used in the [Snap framework](http://github.com:intelsdi-x/snap).
 
 1. [Getting Started](#getting-started)
   * [System Requirements](#system-requirements)
```
Comment (katarzyna-z) [inline README.md]: Please correct below the following line:
```
 * [Configuration and Usage](configuration-and-usage)
```
You need to add `#` before configuration-and-usage.

File: README.md (hunk 2 above)
Comment (marcin-krolik) [inline README.md]: @katarzyna-z done

File: README.md (hunk 3 above)
Comment (marcin-krolik) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (marcin-krolik) [inline README.md]: done

**Comment by katarzyna-z** [comment]
LGTM

**Pull request merged**